### PR TITLE
Create readme file

### PR DESCRIPTION
**Pull request by IoannisByng**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,14 @@
 # pimcore-sitemap-plugin
```
Comment (SeerUK) [inline README.md]: Can this be:

```
Pimcore Sitemap Plugin
======================
```

File: README.md [hunk 2]
```diff
@@ -1,2 +1,14 @@
...
+# Pimcore Sitemap Plugin Settings 
+1. Set the name of the domain at pimcore system settings below the Website tab
+2. Set up enviroment to Production at pimcore System Settings below the Debug tab
+3. Create new predefined property. Then add this property to every page that you want to exclude from the sitemap 
+    * name => Sitemap: Exclude page
+    * key => sitemap_exclude
+    * type => bool  
+    * value => true
+    * inheritable => yes
+
```
Comment (SeerUK) [inline README.md]: We need a license section:

```
## License

MIT
```

File: README.md [hunk 3]
```diff
@@ -1,2 +1,14 @@
 # pimcore-sitemap-plugin
 XML sitemap generator for Pimcore. Built from the document tree.
+
+# Pimcore Sitemap Plugin Settings
```
Comment (SeerUK) [inline README.md]: This should be a second level header (`##`).

File: README.md [hunk 4]
```diff
@@ -1,2 +1,14 @@
 # pimcore-sitemap-plugin
 XML sitemap generator for Pimcore. Built from the document tree.
+
```
Comment (SeerUK) [inline README.md]: Somewhere here squashed between these things we should have some installation instructions too :D, something like:

```
## Installation

To install the plugin, simply require it with Composer:

'''
$ composer require byng/pimcore-sitemap-plugin
'''
```

Make the `'''` bits actual backticks though, like ```, it's meant to be code >_<

**Comment by seeruk** [comment]
LGTM

**Pull request merged**